app/controllers/users_controller.rb

```diff
@@ -1,2 +1,37 @@
 class UsersController < ApplicationController
+  skip_before_action :authenticate_user!, only: :show
```

Comment:
Does this mean I can see a user profile if I am not logged in? I would not allow that